packages/manager/modules/pci/src/components/project/users/users.html

```diff
@@ -133,6 +133,16 @@
                     data-translate="pci_projects_project_users_modify_password_label"
                 ></span>
             </oui-action-menu-item>
+
+            <oui-action-menu-item
+                data-on-click="$ctrl.generateCredentials($row)"
+                data-disabled="$ctrl.isActionDisabled()"
```

Comment:
i think we can rename like this:
```suggestion
                data-disabled="!$ctrl.generateCredentials()"
```

Reply:
If the `isActionDisabled` function it was already defined, so no issues.
If you can rename it it will be great.
it's up to you @varun257

Reply:
This is a common function used for all options and `isActionDisabled` comes as binding to the component `users`.
This is a reference to if database is active or not.

Reply:
Ok, got it @varun257

Reply:
Since it is used for all actions, it is better not to rename this and keep it as a generic name `isActionDisabled`